.github/workflows/maven-deploy.yaml

```diff
@@ -55,7 +55,7 @@ jobs:
           distribution: 'temurin'
           java-version: '17'
           cache: 'maven'
-          server-id: sonatype-nexus-snapshots
+          server-id: ossrh
```

Comment:
Updated config to better match https://central.sonatype.org/publish/publish-maven/ - not sure why we had two server entries w/ the same credentials.